samples/example-nested-jwm.js

```diff
@@ -0,0 +1,93 @@
+/*
+    The following script generates the example signed JWM featured
+    in the draft RFC
+*/
+const jose = require('jose');
+const base64url = require('base64url');
+const fs = require('fs-extra');
+
+async function main() {
+    //Payload definition
+    const payload = {
+        id: "urn:uuid:ef5a7369-f0b9-4143-a49d-2b9c7ee51117",
+        type: "hello-world-message-type",
+        from: "urn:uuid:8abdf5fb-621e-4cf5-a595-071bc2c91d82",
+        expiry: 1516239022,
+        time_stamp: 1516269022,
+        body: { message: "Hello world!" }
+    };
+
+    //Get sender key
+    const senderJWK = JSON.parse(await fs.readFile("example-keys/example-sender.json"));
+    const jwsKey = jose.JWK.asKey(senderJWK);
+
+    //Sign JWM
+    const jws = new jose.JWS.Sign(payload);
+
+    //Hmm, why is this called recipients? 
+    //Should it not be issuers?
```

Comment:
Yeap I thought the same thing when I saw the api

Reply:
I just copied and pasted what you had, but still think the API was a bit weird.